nixos/modules/hardware/video/backlight/mba6x_bl.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, ... }:
+
+with lib;
+
+let cfg = config.services.mba6x_bl;
+    kernel = config.boot.kernelPackages;
+    modprobe = "${config.system.sbin.modprobe}/sbin/modprobe";
+in
+{
+  options = {
+    services.mba6x_bl = {
+      enable = mkEnableOption "mba6x_bl";
+    };
+  };
```

Comment:
This will appear somewhat cryptic to readers of the man page, is it possible to provide a more descriptive name/docs?